docs/getting-started/index.md

```diff
@@ -38,9 +38,9 @@ The starter application is located in the [desktop-starter](https://github.com/i
 
 > Open your snapshot iModel in the desktop-starter app by clicking the home button in the upper left and clicking "Select snapshot".
 
-### [Follow tutorials to dive deeper into iModel.js]($docs/learning/tutorials/index.md)
+### [Follow tutorials to dive deeper into iTwin.js]($docs/learning/tutorials/index.md)
 
-### [iModel.js Blog](https://medium.com/imodeljs)
+### [iTwin.js Blog](https://medium.com/imodeljs)

```

Comment:
Can you update the blob link?  It's now at `https://medium.com/itwinjs`

Reply:
done